NitroxPatcher/Patches/Dynamic/uSkyManager_SetVaryingMaterialProperties_Patch.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections.Generic;
+using System.Reflection;
+using HarmonyLib;
+using NitroxClient.GameLogic;
+using NitroxModel.Helper;
+using NitroxPatcher.PatternMatching;
+using static System.Reflection.Emit.OpCodes;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+/// <summary>
+/// Prevent uSkyManager from "freezing" the clouds when FreezeTime is active (game paused).
+/// Also sets skybox's rotation depending  on the real server time.
+/// </summary>
+public class uSkyManager_SetVaryingMaterialProperties_Patch : NitroxPatch, IDynamicPatch
+{
+    private static readonly MethodInfo TARGET_METHOD = Reflect.Method((uSkyManager t) => t.SetVaryingMaterialProperties(default));
+
+    public static readonly InstructionsPattern ModifyInstructionPattern = new()
+    {
+        Ldarg_0,
+        Ldfld,
+        { Call, "Modify" },
```

Comment:
Having a comment here on what it matches for the call would be nice. Or changing the code to:
```cs
{ InstructionPattern.Call(nameof(UnityEngine.Time), $"get_{nameof(UnityEngine.Time.time)}"), "Modify" },
```